docs/guides/src/main/server/logging.adoc

```diff
@@ -7,90 +7,100 @@ title="Configuring logging"
 |%t|Thread name|Renders the thread name.
 |%t{id}|Thread ID|Render the thread ID.
-|%z{<zone name>}|Time zone|Set the time zone of log output to <zone name>.
+|%z{<zone name>}|Time zone|Sets the time zone of log output to <zone name>.
 |====
 
-=== Set the logging format
-To set the logging format for a logged line, build your desired format template using the table above and run the following command:
+=== Setting the logging format
+To set the logging format for a logged line, perform these steps:
+
+. Build your desired format template using the table above
+. Enter the following command:
++
 <@kc.start parameters="--log-format=\"\'<format>\'\""/>
-Be aware that you need to escape characters when invoking commands containing special shell characters such as `;` using the CLI, so you might want to set it in the configuration file instead.
+
+Note that you need to escape characters when invoking commands containing special shell characters such as `;` using the CLI. Therefore, consider setting it in the configuration file instead.
 
 .Example: Abbreviate the fully qualified category name
+
 <@kc.start parameters="\"\'%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{3.}] (%t) %s%e%n\'\""/>
+
 The example above abbreviates the category name, which could get rather long in some cases, to three characters by setting `[%c{3.}]` in the template instead of the default `[%c]`.
 
-== Configuring raw quarkus logging properties
-At the time of writing, the logging features of the quarkus based Keycloak are basic, yet powerful. Nevertheless, expect more to come and feel free to join the https://github.com/keycloak/keycloak/discussions/8870[discussion] at GitHub.
+== Raw quarkus logging properties
+At publication time for this guide, the logging features of the Quarkus-based Keycloak are basic, yet powerful. Nevertheless, expect more to come and feel free to join the https://github.com/keycloak/keycloak/discussions/8870[discussion] at GitHub.
 
-When you need a temporary solution, e.g. for logging to a file or using syslog isntead of console, you can check out the https://github.com/keycloak/keycloak/discussions/8870[Quarkus logging guide]. It is possible to use all properties mentioned there, as long as no other than the base logging dependency is involved. For example it is possible to set the log handler to file, but not to use json output, yet, as you would need to provide another dependency for json output to work.
+If you need a temporary solution, such as for logging to a file or using syslog instead of the console, you can check out the https://github.com/keycloak/keycloak/discussions/8870[Quarkus logging guide]. You might be able to use all properties mentioned there as long as no other than the base logging dependency is involved. For example, you might be able to set the log handler to file, but not to use JSON output yet. You would need to provide another dependency for JSON output to work.
```

Comment:
```suggestion
If you need a temporary solution, such as for logging to a file or using syslog instead of the console, you can check out the https://quarkus.io/guides/logging[Quarkus logging guide]. You are able to use all properties mentioned in the guide as long as no other than the base logging dependency is involved. For example, you are able to set the log handler to file or syslog, but not to use JSON output yet. You would need to provide another dependency for JSON output to work.
```

Reply:
1) Link was wrong, oops. changed it.
2) "might be" implies to me a non-existing insecurity if the described process is possible. "it might be so, it might not."-  It is possible, so just use "are" instead?